dag-pb/src/lib.rs

```diff
@@ -0,0 +1,74 @@
+//! Protobuf codec.
+pub use crate::codec::{PbLink, PbNode};
+use core::convert::TryInto;
+use libipld_base::cid;
+use libipld_base::codec::Codec;
+use libipld_base::error::{BlockError, IpldError};
+use libipld_base::ipld::Ipld;
+use thiserror::Error;
+
+mod codec;
+
+/// Protobuf codec.
```

Comment:
Could link to spec here https://github.com/ipld/specs/blob/master/block-layer/codecs/dag-pb.md